node_modules/@okta/i18n/package.json

```diff
@@ -1,30 +1,31 @@
 {
   "_args": [
     [
-      "@okta/i18n@28.25.0",
+      "@okta/i18n@30.2.0",
       "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@28.25.0",
-  "_id": "@okta/i18n@28.25.0",
+  "_from": "@okta/i18n@30.2.0",
+  "_id": "@okta/i18n@30.2.0",
   "_inCache": true,
+  "_installable": true,
   "_location": "/@okta/i18n",
   "_phantomChildren": {},
   "_requested": {
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@28.25.0",
-    "rawSpec": "28.25.0",
+    "raw": "@okta/i18n@30.2.0",
+    "rawSpec": "30.2.0",
     "scope": "@okta",
-    "spec": "28.25.0",
+    "spec": "30.2.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```

Comment:
@hor-kanchan-okta not sure I need to check. This file was auto updated using a command.